aiida_restapi/routers/nodes.py

```diff
@@ -73,35 +74,16 @@ async def create_upload_file(
 ) -> models.Node:
     """Endpoint for uploading file data"""
     node_dict = params.dict(exclude_unset=True, exclude_none=True)
-    node_type = node_dict.pop("node_type", None)
-
-    entry_point_node = _get_entry_point(group="aiida.rest.post", name=node_type)
-
-    orm_object = entry_point_node.load().create_new_node_with_file(
-        node_type, node_dict, upload_file
-    )
-
-    return models.Node.from_orm(orm_object)
+    entry_point = node_dict.pop("entry_point", None)
 
-
-def _get_entry_point(group: str, name: str) -> EntryPoint:
-    """Fetch entry point.
-
-    Provides a more user-friendly error message if the entry point is not found.
-    """
-    eps = ENTRY_POINTS.select(
-        group=group,
-        name=name,
-    )
-    if not eps:
-        raise HTTPException(
-            status_code=404,
-            detail=f"Entry point '{name}' not found in group '{group}'.",
-        )
-    if len(eps) > 1 and len(set(ep.value for ep in eps)) != 1:
+    try:
+        cls = load_entry_point(group="aiida.data", name=entry_point)
+    except EntryPointError as exception:
         raise HTTPException(
             status_code=404,
-            detail=f"Multiple entry points '{name}' found in group '{group}': {eps}",
-        )
+            detail=f"Could not load entry point from group `aiida.data`: {exception}",
```

Comment:
```suggestion
            detail=f"Could not load entry point '{entry_point}' from group `aiida.data`: {exception}",
```

Reply:
In fact, this information (as well as the group) is already included in the `{exception}` so I think it is better to even remove `from group aiida.data` as well.